### ISSUE TITLE
[ART]Isometric vs Top Down vs Fully Flat

### ISSUE THREAD

**Issue by longtomjr** [TeamPorcupine/ProjectPorcupine#1081]
This is a discussion thread for Isometric vs Topdown vs Fully Flat
*Examples*
### Top Down (aka angled Top Down)
![](http://liza.io/images/2012/02/creatures-and-castles.png)
### Isometric
![](http://1.bp.blogspot.com/--w3dyXRR5pA/Tt6fXQCDs2I/AAAAAAAABDw/UwFy8XUg9Tw/s1600/display_1.jpg)
### Fully Flat (aka true Top Down)
![](http://1.bp.blogspot.com/-DdgIcm5sohI/UY0vkAR2m9I/AAAAAAAALF8/sP3WdN-7AWw/s1600/Monaco-2.jpg)

**Comment by Tranberry** [comment]
I love isometric, but that would mean that I'm out. I can't make that kind of art (I won't have the time to do that). I vote for keeping it top down at an angle, else you have to suffer my code. 

Another reason to keep it top-down is that most people can make pretty decent sprites that way, imo.

**Comment by longtomjr** [comment]
Here is my 2 cents:  
**TopDown:** Prolly the best for this project. Having things at an angle helps to convey more info about that object and it is simple and clean. This is also the easiest to draw for and have shown success in the building game scene.  
**Isometric:** Hard, really hard. Looks great but I don't think it will work for this project. There will need to be camera changes and being in space will make it necessary to render multiple levels. This would be great if we were a full time studio with dedicated artists, but for this project I don't think it will work.  
**Fully Flat:** This is also a really hard style to pull of because you are limited from one angle. Another problem is that it might be a lot more difficult to convey the necessary info when you can only draw one side.  

**TLDR**(*though you should really read my thoughts)*: I am going for TopDown!

**Comment by longtomjr** [comment]
also, just some pings...  
@ComputerOverlord, @Mizipzor , @dusho , @Tranberry , @longtomjr , @abackwood @alexisfcote

**Comment by Tranberry** [comment]
@TheRiven ^ @longtomjr was lazy.

**Comment by TheRiven** [comment]
That isometric pic looks awesome :+1: 

But I do feel top-down is a good look for us to continue with, although the fully flat could probably work as well. :thought_balloon:

**Comment by Grenkin1988** [comment]
**Isometric** is so beautiful. But it would be hell to do it.
**Top down** I think

**Comment by vogonistic** [comment]
I don't think this should be an isometric game. 

The only worry I have between Top Down and Fully Flat is that there is a perspective in Top Down that means we need 4x the number of sprites if we want to rotate furniture.

**Comment by Tranberry** [comment]
or be lazy:

![mockup](https://cloud.githubusercontent.com/assets/16529/18183089/09848c36-7093-11e6-946b-414c4407544d.png)

**Comment by Tranberry** [comment]
ping @Raketfart @Tritaris anyone with the least bit of interest in sprites?

**Comment by dusho** [comment]
top down
also, does that mean that there shouldn't be light applied to sprites (from N)

**Comment by Tranberry** [comment]
no it does not, as far as I'm concerned at least.

**Comment by Raketfart** [comment]
:+1: Vote for top down zelda style. 
All flat is hella hard to do if the player has to be able to tell what it is.

I think we'll have some sprites that don't require rotation, like the cloning machine. Some sprites will require a vert (+ flipped) and horizontal version. And then finally some will have to have 4 orientations. 

Keeping rotation in mind when designing a piece of furnite will be helpful in cutting down on required rotations.

**Comment by Sommerbo** [comment]
3/4 isometric is awesome but very labor intensive, and wouldn't fit our game
Flat lacks a lot of contextual clues and can be confusing. DF uses this
Top-down is the only sensible choice I feel. Used by PA and Rimworld

**Comment by Tritaris** [comment]
my vote is top down. its a nice bridge between flat and isometric.

**Comment by ComputerOverlord** [comment]
SOLUTION:
![iso](https://cloud.githubusercontent.com/assets/21087045/18186469/cdc96e54-7057-11e6-9d5e-7612adcbc59b.PNG)

**Comment by Yurgeta** [comment]
flat is quicker and easier and can contain detailed work all the same as 3/4 except you just dont get a side view of things aswell

isometric is a pain to do and even more work required to get things looking right especially for circular objects.

3/4 perspective is easy as well and can be a lot more pleasing to the eye but there are important issues needed to be taken into consideration that are key to pulling it off..

**Comment by powli** [comment]
I think both, flat and angled top down would work really well for this project.

Isometric not only requires more work, but makes it harder to get the controls right from a gameplay perspective.

**Comment by sboigelot** [comment]
As @powli I think flat or angled top down would be better.

Isometric also make mod author life more complicate. It is easier for non-artist to make a flat image with paint, hard to adapt perspective for a good isometric view.

**Comment by dusho** [comment]
can we get then basic primitive shapes (cube, sphere?, ...) from some artists under right angle? so we will have consistent angles and have dimensions right
image from @Tranberry looks awesome.. can we use it as some baseline ?

**Comment by ajahraus** [comment]
I love the look of Isometric, but I think that's too a bit too tough. Fully-Flat is simple, but it makes things hard to distinguish. I certainly vote angled Top-down, although that means that there will probably need to be additional art for lots of sprites, since often rotating doesn't make sense.

**Comment by ComputerOverlord** [comment]
I like how everyone ignored my post :laughing: 
So... I guess that would suck because... 3D Modelling? I made a couple models that looked sort of cool last night.. and I don't even have much experience!

**Comment by ComputerOverlord** [comment]
Top down is RPG topdown, not our current top down

**Comment by Tranberry** [comment]
@ComputerOverlord please post these models, and maybe be tiny bit more descriptive. I didn't get what that solution was.

**Comment by bjubes** [comment]
I vote top down

**Comment by frankitox16** [comment]
I'd go for top down because of the simplicity of it and how usefull it can be. It's easier to tell differences of furniture this way, and (although we have to make more sprites) it's more "descriptive" imo.

**Comment by longtomjr** [comment]
I am closing this Issue. We are now gonna use top down (since that is the consensus here).

Thanks for participating!
